### PR TITLE
Merge 1.1 into 2.0

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: bindings/core/src/method_handler/client.rs [hunk 1]
```diff
@@ -168,7 +168,7 @@ pub(crate) async fn call_client_method_internal(client: &Client, method: ClientM
         }
         ClientMethod::GetNode => Response::Node(client.get_node().await?),
         ClientMethod::GetNetworkInfo => Response::NetworkInfo(client.get_network_info().await?),
-        ClientMethod::GetNetworkId => Response::NetworkId(client.get_network_id().await?),
+        ClientMethod::GetNetworkId => Response::NetworkId(client.get_network_id().await?.to_string()),
```
Comment (DaughterOfMars) [inline bindings/core/src/method_handler/client.rs:171]: Is this needed?

**Review by Thoralf-M — APPROVED** [review]

File: bindings/core/src/method_handler/client.rs (hunk 1 above)
Comment (thibault-martinez) [reply bindings/core/src/method_handler/client.rs:171]: Yes

File: bindings/core/src/response.rs [hunk 2]
```diff
@@ -68,7 +68,7 @@ pub enum Response {
     NetworkInfo(NetworkInfo),
     /// Response for:
     /// - [`GetNetworkId`](crate::method::ClientMethod::GetNetworkId)
-    NetworkId(u64),
+    NetworkId(String),
```
Comment (DaughterOfMars) [inline bindings/core/src/response.rs:71]: pour quoi non `#[serde(with = "string")]`

Comment (thibault-martinez) [reply bindings/core/src/response.rs:71]: @Thoralf-M Would that also work? But anyway, this would need to be changed in develop, not here.

Comment (Thoralf-M) [reply bindings/core/src/response.rs:71]: Yeah don't see why not

File: bindings/python/examples/wallet/offline_signing/1_prepare_transaction.py [hunk 3]
```diff
@@ -27,7 +27,8 @@
 
 # Recovers addresses from example `0_address_generation`.
 addresses_data = json.load(open(ADDRESSES_FILE_PATH, "r", encoding="utf-8"))
-addresses = list(map(lambda x: from_dict(data_class=AccountAddress, data=x), addresses_data))
+addresses = list(map(lambda x: from_dict(
```
Comment (DaughterOfMars) [inline bindings/python/examples/wallet/offline_signing/1_prepare_transaction.py:30]: :c

**Review by DaughterOfMars — APPROVED** [review]

File: bindings/core/src/response.rs (hunk 2 above)
Comment (DaughterOfMars) [reply bindings/core/src/response.rs:71]: 👍

**Review by kwek20 — APPROVED** [review]

**Pull request merged**